azurerm/internal/services/azurestackhci/registration.go

```diff
@@ -0,0 +1,29 @@
+package azurestackhci
+
+import "github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+
+type Registration struct{}
+
+// Name is the name of this Service
+func (r Registration) Name() string {
+	return "Azure Stack HCI"
+}
+
+// WebsiteCategories returns a list of categories which can be used for the sidebar
+func (r Registration) WebsiteCategories() []string {
+	return []string{
+		"Azure Stack HCI",
+	}
+}
+
+// SupportedDataSources returns the supported Data Sources supported by this Service
+func (r Registration) SupportedDataSources() map[string]*schema.Resource {
+	return map[string]*schema.Resource{}
+}
+
+// SupportedResources returns the supported Resources supported by this Service
+func (r Registration) SupportedResources() map[string]*schema.Resource {
+	return map[string]*schema.Resource{
+		"azurerm_hci_cluster": resourceArmHCICluster(),
```

Comment:
this name isn't clear this is for Azure Stack HCI directly, should this become `azurerm_stack_hci_cluster`?

Reply:
updated